src/cli/commands/add.js

```diff
@@ -107,7 +107,22 @@ export class Add extends Install {
 
   async init(): Promise<Array<string>> {
     this.addedPatterns = [];
-    const patterns = await Install.prototype.init.call(this);
+    let patterns = [];
+    try {
+      patterns = await Install.prototype.init.call(this);
+    } catch (e) {
+      if (e instanceof PackageVersionError) {
+        const replace = this.args.find((pattern) => pattern.indexOf(e.packageName) > -1);
+        if (replace) {
+          this.args.splice(this.args.indexOf(replace), 1, `${e.packageName}@${e.resolvedVersion}`);
+          patterns = await Install.prototype.init.call(this);
```

Comment:
I'm catch new `PackageVersionError` here and restarting the install process with the resolved value from the inquirer.